src/SDK/Trace/Span.php

```diff
@@ -249,6 +251,15 @@ public function end(int $endEpochNanos = null): void
             return;
         }
 
+        $spanData = $this->toSpanData();
+        if ($spanData->getTotalDroppedLinks() || $spanData->getTotalDroppedEvents() || $spanData->getAttributes()->getDroppedAttributesCount()) {
+            self::logWarning('Dropped span attributes, links or events', [
```

Comment:
Having the trace and span IDs also logged could help people find the actual traces/spans where this happened. While the actual dropped links/events/attributes would obviously not be recorded there, at least it gives more context.